match/lib/match/runner.rb

```diff
@@ -71,6 +71,51 @@ def run(params)
       GitHelper.clear_changes
     end
 
+    def import_certificate(args, params)
+      UI.user_error!("It's not allowed to import your certificate because you enabled `readonly` option.") if params[:readonly]
+      UI.warning("Enterprise profiles are currently not officially supported in _match_, you might run into issues") if params[:type] == "enterprise"
+
+      params[:import_certificate] = args[0] if params[:import_certificate].nil?
+      UI.user_error!("Missing path to the certificate to import") if params[:import_certificate].nil?
+
+      params[:import_certificate_password] = args[1] if params[:import_certificate_password].nil?
```

Comment:
@razum2um Hey, so currently:

if you ran:
```
MATCH_IMPORT_CERTIFICATE=/path_to_cert fastlane match import $CERTIFICATE_PASSWORD
```

it would fail to find the password, and could be somewhat confusing. This is however, potentially fine, I was mostly interested to see what your thoughts were and if you'd considered it.